### ISSUE TITLE
White loading screen when opening deck options (Qt6)

### ISSUE THREAD

**Issue by Andrew15-5** [ankitects/anki#1910]
A couple of days ago I installed the latest version of  Anki Qt6 on Ubuntu 20.04 and among some small things the huge one is that every single time when I need to change deck options (any preset) I have to see a shmicking flashbang in my face. And by default, Anki (besides the initial white windows) is using a pretty nice dark theme.
And why new options menu is loading anyway (for about 1 second)? The legacy menu (Shift+Click) shows up instantly, but doesn't have the new V3 scheduler submenu (otherwise I would probably stick to it, instead of the new one).

**Comment by Andrew15-5** [comment]
Ok, today I tried to install Qt5 version. The loading screen (still about 1 second long) is still present, but its color is correct - "as all things should be" (dark theme #2F2F31 ![image](https://user-images.githubusercontent.com/37143421/173589621-2573494a-c120-4102-8b79-cb3e39efc376.png)).
So, Qt6 isn't perfect after all, that's understandable (it was somewhat recently released).

If you don't fix this in Qt6, then you can close this issue.

**Comment by Andrew15-5** [comment]
Today I encountered the white background during loading, but only once (Qt5). I think it's due to the size of the deck (pretty big; the options of which I was opening) or because of some initial stuff happening behind the scene (initialization of some sort).

**Comment by dae** [comment]
Is this still an issue with the latest Anki Qt6 version?

**Comment by Andrew15-5** [comment]
No. Now the option window starts with a black background... most of the time (about 90%), but a few times it starts with the dark theme background (which is more preferable IMO).
